alibi_detect/cd/keops/learned_kernel.py

```diff
@@ -130,7 +131,7 @@ def __init__(
             input_shape=input_shape,
             data_type=data_type
         )
-        self.meta.update({'backend': 'keops'})
+        self.meta.update({'backend': Framework.KEOPS})
```

Comment:
Even though for `str enums` in a python interpreter `Framework.KEOPS == 'keops'` is `True`, I believe for user facing string values we need to do `Framework.KEOPS.value`...

Reply:
That being said `json.dumps({'backend': Framework.KEOPS})` does the right thing, so perhaps this is mostly a question of representing the metadata visually in the Python interpreter.

Reply:
I think you're right, this was actually why I changed to `Framework.SKLEARN.value` in https://github.com/SeldonIO/alibi-detect/pull/650#discussion_r994429416, as the `Framework.SKLEARN` was causing the pydantic validation to fail. I (think) I agree that anything public facing should have a `.value`.

Reply:
Changed to all `self.meta` updates to use `.value` in 44a7200.